### PR TITLE
Fix submodule path in gradle for commit hash check

### PR DESCRIPTION
**Pull request by usmansaleem**
## PR description
Fix submodule path in gradle for hash check

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

Signed-off-by: Usman Saleem <usman@usmans.info>

### CODE REVIEW

File: ethereum/referencetests/build.gradle [hunk 1]
```diff
@@ -55,7 +55,7 @@ task ('validateReferenceTestSubmodule') {
     def submoduleHashes = [
       [
         '2fe00d76d97e35aa2df6449b5f567c41cf6f58fc',
```
Comment (jframe) [inline ethereum/referencetests/build.gradle:57]: Why is this the qbft reference test submodule hash even being checked as part of the ethereum reference tests module? Don't think this should be here at all. Can we change the command to only look for the ethereum/referencetests git submodule?

Comment (shemnon) [reply ethereum/referencetests/build.gradle:57]: The change that added the QBFT reference tests undid the check for the ethereum reference tests.  We need to make sure that the reference tests are not accidentally changed via git merges.  Which happens very frequently after updates (one which just happened)

Comment (shemnon) [reply ethereum/referencetests/build.gradle:57]: Perhaps we should update line 66 as well and add a third "human readable" description of the sub modeile, as well as line 69.  This should improve clarity.

Comment (usmansaleem) [reply ethereum/referencetests/build.gradle]: I've just pushed another fix ... each submodule respectively checks its own hash.

Comment (jframe) [reply ethereum/referencetests/build.gradle]: That's a nicer fix

**Review by jframe — APPROVED** [review]

**Review by macfarla — APPROVED** [review]
LGTM

**Pull request merged**